src/uct/api/v2/uct_v2.h

```diff
@@ -149,8 +149,13 @@ typedef enum {
  * @ingroup UCT_MD
  * @brief Completion callback for memory region invalidation.
  *
- * This callback routine is invoked when when it is no longer accessible by
+ * This callback routine is invoked when it is no longer accessible by
  * remote peer.
+ * 
+ * $note: in some implementations this callback may be called immediately after
+ *        @ref uct_md_mem_dereg_v2 is called, but it is possible that callback
```

Comment:
that the callback call will be delayed until all references to the memory region, for example, when ... in the implementation of the memory domains.